tests/test_cargo_generate_lockfile.rs

```diff
@@ -171,3 +171,26 @@ test!(preserve_line_endings_issue_2076 {
     assert!(lock2.starts_with("[root]\r\n"));
     assert_eq!(lock1, lock2);
 });
+
+test!(cargo_update_generate_lockfile {
+    let p = project("foo")
+        .file("Cargo.toml", r#"
+            [package]
+            name = "foo"
+            authors = []
+            version = "0.0.1"
+        "#)
+        .file("src/main.rs", "fn main() {}");
+
+    let lockfile = p.root().join("Cargo.lock");
+    assert_that(&lockfile, is_not(existing_file()));
+    assert_that(p.cargo_process("update"), execs().with_status(0));
```

Comment:
Can you add `.with_stdout` here to ensure that nothing spurious is getting printed (same below as well)